src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -183,7 +184,7 @@ const GovernanceReviewTeam = () => {
               render={() => (
                 <SubmitAction
                   action="not-an-it-request"
```

Comment:
Is it preferable to have separate routes for each action selected? Or is there a way to chain the action select page and the feedback page into a single form?

Reply:
Chaining into the same form gets complicated and doesn't allow a user to use their native back button. We also might lose their status on refresh. It's usually better to have a named route for each section of the form.

However, we can do some code sharing because each action renders the same component, passing it the pieces that are different for that specific action.